specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json

```diff
@@ -1,3091 +1,3091 @@
+                  "configMediaType": "application/vnd.docker.container.image.v1+json",
+                  "tags": [
+                      "4.7.2-20180905-nanoserver-1803"
+                  ],
+                  "changeableAttributes": {
+                      "deleteEnabled": true,
+                      "writeEnabled": true,
+                      "readEnabled": true,
+                      "listEnabled": true
+                  }
+              }
           }
-        ],
-        "responses": {
-          "201": {
-            "description": "The upload has been completed and accepted by the registry.",
-            "headers": {
-              "Location": {
-                "description": "Provided location for blob",
-                "type": "string"
-              },
-              "Range": {
-                "description": "Range indicating the current progress of the upload.",
-                "type": "string"
-              },
-              "Docker-Content-Digest": {
-                "description": "Digest of the targeted content for the request.",
-                "type": "string"
-              }
-            }
+      },
+      "ManifestAttributesBase": {
+          "required": [
+              "digest",
+              "createdTime",
+              "lastUpdateTime"
+          ],
+          "type": "object",
+          "description": "Manifest details",
+          "x-accessibility": "internal",
+          "properties": {
+              "digest": {
+                  "type": "string",
+                  "readOnly": true,
+                  "description": "Manifest"
+              },
+              "imageSize": {
+                  "type": "integer",
+                  "readOnly": true,
+                  "format": "int64",
+                  "description": "Image size",
+                  "x-ms-client-name": "size"
+              },
+              "createdTime": {
+                  "type": "string",
+                  "readOnly": true,
+                  "format": "date-time",
+                  "description": "Created time",
+                  "x-ms-client-name": "createdOn"
+              },
+              "lastUpdateTime": {
+                  "type": "string",
+                  "readOnly": true,
+                  "format": "date-time",
+                  "description": "Last update time",
+                  "x-ms-client-name": "lastUpdatedOn"
+              },
+              "architecture": {
+                  "$ref": "#/definitions/ArtifactArchitecture",
+                  "description": "CPU architecture",
+                  "readOnly": true,
+                  "x-nullable": true
+              },
+              "os": {
+                  "$ref": "#/definitions/ArtifactOperatingSystem",
+                  "description": "Operating system",
+                  "x-ms-client-name": "operatingSystem",
+                  "readOnly": true,
+                  "x-nullable": true
+              },
+              "mediaType": {
```

Comment:
Is it alright to add this here?

Reply:
Yes, we should add. I don't think we need to make it required because it is only returned in response. Not something user can input.